src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Cvt.kt

```diff
@@ -52,6 +55,7 @@ class Cvt : AbstractCommand(
             channel.sendMessage(
                 "Temperature units to convert to are `${temps.joinToString("`, `")}` from those values.\n" +
                     "Height units to convert to are `${lengths.joinToString("`, `")}` from those same values as well.\n" +
+                    "Weight units to convert to are `${weights.joinToString("`, `")}` again from the same values.\n" +
                     "The syntax is `${Yuuto.config["PREFIX"]}cvt <unit-to-convert-to> <value>`"
```

Comment:
User probably already knows the prefix, given that they've actually run the command. Usage instructions shouldn't contain the prefix anyway.

Reply:
I disagree since users are idiots. If they run a command that misses arguments and you tell them to type some words, they will type words in the chat.